pages/common/ngrep.md

```diff
@@ -0,0 +1,23 @@
+# ngrep
+
+> Grep traffic on a network.
```

Comment:
Instead of "Grep", which is a specialized/jargon term, I'd suggest something like "Filter network traffic packets using regular expressions." (or "Search" instead of "Filter", if you prefer)